synapse/rest/client/v2_alpha/relations.py

```diff
@@ -243,6 +249,12 @@ def on_GET(self, request, room_id, parent_id, relation_type=None, event_type=Non
         from_token = parse_string(request, "from")
         to_token = parse_string(request, "to")
 
+        # Check if the event is redacted, and if so return an empty chunk
+        # list and zero tokens
+        if event.internal_metadata.is_redacted():
+            res = {"chunk": []}
```

Comment:
it might be nice to construct an empty PaginationChunk here, so that we know the behaviour wrt pagination tokens is the same as the other code path?